lib/data_migrate/database_tasks.rb

```diff
@@ -68,28 +68,23 @@ def run_migration(migration, direction)
         end
       end
 
-      def schema_dump_path(db_config, format = ActiveRecord.schema_format)
-        return ENV["DATA_SCHEMA"] if ENV["DATA_SCHEMA"]
+      unless Gem::Dependency.new("railties", "< 7.0").match?("railties", Gem.loaded_specs["railties"].version)
```

Comment:
I'd change the `unless` to an `if` and flip the condition, I think it makes it easier to read